### PR TITLE
Two more questions from Nick's Twitter quiz

### PR DESCRIPTION
**Pull request by jamesleesaunders**
Adding two more questions from Nick's Twitter quiz:
* Reduce Array from 2019-09-22.
* Console Log Constructors from 2019-09-19.

Towards #25

### CODE REVIEW

File: content/questions/reduce-array/index.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+const result = Object
+  .values(scrambled)
+  .reduce((agg, el) => agg + el, "");
+
+console.log(result);
+```
+
+<!-- explanation -->
+
+The reduce array method takes two arguments: a reducer function and an initial value.
```
Comment (nas5w) [inline content/questions/reduce-array/index.md]: Can you additionally add that `Object.values` iterates through enumerable number keys in sequential order? This is why the letters appear to sort themselves correctly.

File: content/questions/reduce-array/index.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+title: Reduce Array
```
Comment (nas5w) [inline content/questions/reduce-array/index.md]: Can we change this to reduce-object? Perhaps rename the file too?

File: content/questions/reduce-array/index.md (hunk 1 above)
Comment (jamesleesaunders) [reply content/questions/reduce-array/index.md]: No problem! Done!

File: content/questions/reduce-array/index.md (hunk 2 above)
Comment (jamesleesaunders) [reply content/questions/reduce-array/index.md]: Now renamed.

**Comment by jamesleesaunders** [comment]
Thanks Nick.
Suggested changes made.

**Pull request merged**